### PR TITLE
twis62

### PR DESCRIPTION
**Pull request by larsbergstrom**
r? @jdm

---

This change is [Reviewable](https://reviewable.io/reviews/servo/blog.servo.org/82)

### CODE REVIEW

File: _posts/2016-05-09-twis-62.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Our overall [roadmap](https://github.com/servo/servo/wiki/Roadmap) and [quarterly goals](https://docs.google.com/document/d/1JMOtVkRtb-s7auoQdnX810HGglkMK054LTXOo0_rdrU/pub) are available online.
+
+This week's status updates are [here](http://statusupdates.dev.mozaws.net/project/servo).
+
+### Notable Additions
+ 
+ - ms2ger [added](https://github.com/servo/rust-selectors/pull/86) support for using Gecko's string atoms in rust-selectors, which is a major performance issue for reusing Servo's style system in Firefox
+ - autrilla [changed](https://github.com/servo/servo/pull/11067) our build driver script, `mach`, to display more information when there is a virtualenv or python failure
+ - larsberg [increased](https://github.com/servo/saltfs/pull/366) the number of parallel processes in use on our Mac builders which, in conjunction with some other system tuning, reduces the OSX builder times to around 30 minutes
```
Comment (cbrewster) [inline _posts/2016-05-09-twis-62.md:21]: servo/saltfs#367 reverted this

Comment (larsbergstrom) [inline _posts/2016-05-09-twis-62.md:21]: Only the test-css portion, not the test-wpt portion. test-wpt is where we saw significant speedups; test-css only got us a couple of minutes.

Comment (cbrewster) [inline _posts/2016-05-09-twis-62.md:21]: Ahh I see, my bad!

File: _posts/2016-05-09-twis-62.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ - mbrubeck [fixed](https://github.com/servo/servo/pull/11037) a case where layout could cause more nodes to be laid out than were necessary
+ - glennw [switched](https://github.com/servo/servo/pull/11021) the Android build to officially use OpenGL ES3 now that the corresponding WebRender support landed
+ - heycam [added](https://github.com/servo/servo/pull/11002) support for `text-transform` in geckolib
+ - mmatyas [upgraded](https://github.com/servo/servo/pull/10990) Cargo, to pick up much better support for per-target build configurations
+ - nox [landed](https://github.com/servo/servo/pull/10756) an updated to SpiderMonkey!
```
Comment (asajeffrey) [inline _posts/2016-05-09-twis-62.md:26]: Typo: "an updated" should be "an update".

File: _posts/2016-05-09-twis-62.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ - dati [updated](https://github.com/servo/servo/pull/10632) the WebBluetooth implementation significantly to pick up spec changes
+ - jdm [implemented](https://github.com/servo/servo/pull/8454) basic `<media>` infrastructure
+ - aneeshusa [made](https://github.com/servo/saltfs/pull/298) the Vagrant support for testing our CI builders more full-featured
+ - connorgbrewster [corrected](https://github.com/servo/servo/pull/11020) some mistakes that prevented logging into sites like Twitter and Github
+ -  pcwalton [rewrote](https://github.com/servo/servo/pull/10691) significant parts of float-related layout to be more correct
+ -  nox [upgraded](https://github.com/servo/servo/pull/10756) SpiderMonkey to use version 46
```
Comment (asajeffrey) [inline _posts/2016-05-09-twis-62.md:32]: Duplicate entry.

**Pull request merged**